OrbitCore/LinuxTracingHandler.cpp

```diff
@@ -33,6 +36,7 @@ void LinuxTracingHandler::Start() {
   tracer_->SetTraceContextSwitches(GParams.m_TrackContextSwitches);
   tracer_->SetTraceCallstacks(true);
   tracer_->SetTraceInstrumentedFunctions(true);
+  tracer_->SetTraceGpuDriver(absl::GetFlag(FLAGS_trace_gpu_driver_events));
```

Comment:
Nit: For consistency I would either rename the flag (fine with me) or the member and function: "trace gpu driver" vs. "trace gpu driver events".

Reply:
Agree, I renamed it to trace_gpu_driver. It should make for a better name for a flag.